testing/server.test.js

```diff
@@ -0,0 +1,37 @@
+const request = require('supertest');
+const server = 'http://localhost:3000'
+const app = require('../server/server');
+
+
+describe('verifyUser', () => {
+  describe('/', () => {
+    describe('/login', () => {
+      it('responds with 200 status and text/html content type', () => {
+        return request(app)
+          .post('/login')
+          .expect('Content-Type', /application\/json/)
+          .expect(200);      
+      })
+    });
+  });
+});
+
+describe('searchYelp', () => {
+    describe('GET', () => {
+      it('responds with an empty json object if name/zip are undefined', () => {
```

Comment:
Update this test description to match the test (responds with 404… not clear what the response body is from the test itself).

Should also probably test expect assertion for missing name **and** missing zip